ledger-service/http-json/BUILD.bazel

```diff
@@ -472,6 +625,7 @@ alias(
         deps = [
             ":http-json-{}".format(edition),
             ":integration-tests-lib-{}".format(edition),
+            ":integration-tests-lib-pkg-{}".format(edition),
```

Comment:
Are we now compiling the integration tests 4 times instead of 2?

If so, shouldn't this dep be removed?

If not, do the tests run with this arrangement? (Since the `_suite` variant behaves quite differently from the ordinary variant.)